evansmusomi/react-native-102
# Removes Image from Cloud Storage when Place is Deleted

*What this does*
- Cleans file storage when a place is deleted

*How to verify*
- Check the number of images on Firebase Storage
- Delete a place
- The number of images on Firebase Storage should be less one
